rust/arrow/src/csv/reader.rs

```diff
@@ -87,19 +87,19 @@ fn infer_field_schema(string: &str) -> DataType {
 /// with `max_read_records` controlling the maximum number of records to read.
 ///
 /// If `max_read_records` is not set, the whole file is read to infer its schema.
-fn infer_file_schema<R: Read + Seek>(
+pub fn infer_file_schema<R: Read + Seek>(
     reader: &mut BufReader<R>,
     delimiter: u8,
     max_read_records: Option<usize>,
-    has_headers: bool,
```

Comment:
Should we change `has_headers` to `has_header` consistently on the rest of the `csv` submodule?

Reply:
yeah, i have been thinking about that as well, @andygrove what do you think?

Reply:
That would be great.